azure/vm.tf

```diff
@@ -103,14 +103,14 @@ resource "azurerm_linux_virtual_machine" "crate" {
   source_image_reference {
     publisher = "canonical"
     offer     = "ubuntu-24_04-lts"
-    sku       = "server-gen1"
+    sku       = "server"
     version   = "latest"
   }
 
   os_disk {
     name                 = "OSdisk-crate-${count.index}"
     caching              = "ReadWrite"
-    storage_account_type = "Standard_LRS"
+    storage_account_type = "StandardSSD_LRS"
```

Comment:
There are a bunch of other updates due. For example, `storage_account_type` for the data disk currently uses `Premium_LRS` but there is already `PremiumV2_LRS` available. The default VM instance type is also several generations behind. I tried bumping all of them to the latest and greatest, but ran into errors.

If we ever want to use the Azure Terraform for serious benchmarks, a more in-depth review of all such settings is due. But at least it's now a little less behind.